src/adapters/postgres/user-adapter.ts

```
@@ -1991,6 +1994,21 @@ export class PostgresUserService implements IServicelocator {
+      let referLinkForElastic: string | null = null;
+      try {
+        const incomingSlug = (userCreateDto as any)?.referralSlug ?? null;
+        const attr = await this.referralsService.createUserAttribution({
+          userId: result.userId,
+          incomingSlug,
+        });
+        referLinkForElastic =
+          attr.referLink ??
+          (attr.resolvedSlug ? buildReferLink(attr.resolvedSlug) : null);
+      } catch (refError: any) {
+        LoggerUtil.error('Referral attribution failed', refError);
```

ESLint

[error] Replace `'Referral·attribution·failed'` with `"Referral·attribution·failed"`

(prettier/prettier)